### PR TITLE
Dragging people across the floor now gives you big red message that tells you you're not doing jack shit for them except making them hurt more

### PR DESCRIPTION
**Pull request by astralloop**
Meanwhile, on on today's episode of 'closed PRs with features that make sense'....

:cl: Astral
tweak: Dragging people across the floor now hurts them more. Maybe you should use the fucking roller beds that were made for this bullshit?
/:cl:

### CODE REVIEW

**Comment by ishitbyabullet** [comment]
Salt.

In all seriousness, unless I'm MD, I'm not gonna powergame a fucking roller bed just so I can not get my chat spammed with text.

**Comment by astralloop** [comment]
@ishitbyabullet the semi-spam is meant to tell you "HEY YOU'RE FUCKING KILLING THE GUY FASTER YOU DIPSHIT"

**Comment by astralloop** [comment]
Or, ya know, get a locker or crate

**Comment by ishitbyabullet** [comment]
@astralloop imagine this
>see someone in crit about to get sucked into a plasma fire
>drag hi-
**>**WEE WOO STOP DRAGGING HIM WITHOUT A ROLLER BED YOU FAT FUCK WEE WOO****
>just let him fucking die so i don't get spammed with text

**Comment by astralloop** [comment]
@ishitbyabullet try a locker or crate, they're dime-a-dozen

**Comment by arsserpentarium** [comment]
Not bad idea. But maybe you chould think more about formula.
Because it should work mostly for guys with bruteloss. And bruteloss bigger than some threshold.

**Comment by ExcessiveUseOfCobblestone** [comment]
Another “it’s the good kind of spam guys!” Pr my favorite.

Can’t wait for “Not using the most viable weapon within 3 tiles now spams your chat” and it’s sister “taking damage now spams your chat telling you to get the fuck out”

File: code/modules/mob/living/living.dm [hunk 1]
```diff
@@ -465,8 +464,11 @@
 	if (s_active && !(CanReach(s_active,view_only = TRUE)))
 		s_active.close(src)
 
-	if(lying && !buckled && prob(getBruteLoss()*200/maxHealth))
-		makeTrail(newloc, T, old_direction)
+	on_drag(newloc, old_direction)
+
+/mob/living/proc/on_drag(newloc, dir)
+	if(prob(getBruteLoss()*200/maxHealth) && lying && !buckled && isturf(loc) && has_gravity())
+		makeTrail(newloc, loc, dir)
```
Comment (ChangelingRain) [inline code/modules/mob/living/living.dm]: you need the old turf or you'll break trails, loc will always be the same as newloc here

**Comment by optimumtact** [comment]
This should warn once upon dragging and then never again

**Comment by ExcessiveUseOfCobblestone** [comment]
Why is this on being dragged and not on the person dragging

**Comment by ExcessiveUseOfCobblestone** [comment]
Or just not have the message because dying to incompetence is part of the !fun! that is ss13

**Comment by davethwave** [comment]
How about when your getting dragged on random items that are on the ground such as ex. metal bars, boxes, syringes.
Also it would be a neat feature that if you fall on or get dragged on a syringe some of the chems get injected into you.

**Comment by Shapsy** [comment]
"You should have to use rollerbeds to move people around because it's thematic that dragging bleeding people on the ground would harm them"

But wait, that's a terrible idea because all of the rollerbeds are located in medbay, the one place that they aren't needed because they're already where they need to be!

"Well then just shove them in a locker or a crate then, totally defeating the thematic purpose of it"

**Comment by GuppyLaxx** [comment]
>"g-guys its ok! just use a roller bed like ur supposed to!! or if you cant just use a 

if you're gonna say "just use a locker" then why are you making this in the first place apart from an unneeded annoyance

**Comment by Buggy123** [comment]
Also the PR title is misleading, this does not "add a message that tells you you're hurting people by dragging them", it **makes it so that dragging people hurts them**. Misleading! Bad! Disappointed!

**Comment by epochayur** [comment]
oh wow, buggy's right

shit title, shit PR

**Comment by optimumtact** [comment]
Huh I thought this was a messaging change but no, this is terrible unless multiple new modes of moving people are added

**Pull request closed without merging**